onedal/datatypes/table.cpp

```diff
@@ -103,9 +103,9 @@ ONEDAL_PY_INIT_MODULE(table) {
         return numpy::convert_to_table(obj, queue);
     });
 
-    m.def("from_table", [](const dal::table& t) -> py::handle {
+    m.def("from_table", [](const dal::table& t) -> py::object {
         auto* obj_ptr = numpy::convert_to_pyobject(t);
-        return obj_ptr;
+        return pybind11::reinterpret_steal<py::object>(obj_ptr);
```

Comment:
```suggestion
        return py::reinterpret_steal<py::object>(obj_ptr);
```
Just a tiny nitpick.